errors/next-prerender-current-time.md

```diff
@@ -27,42 +32,100 @@ After:
 ```jsx filename="app/page.js"
 export default async function Page() {
   const start = performance.now();
-  const data = await somethingSlow();
+  const data = computeDataSlowly(...);
   const end = performance.now();
   console.log(`somethingSlow took ${end - start} milliseconds to complete`)
   return ...
 }
 ```
 
-If you want the current time to be reactive so it can change and update consider moving this usage to a Client Component. Note that Server Side Rendering timestamps is also problematic because hydration will fail so you must only set the time after hydration has completed using `useEffect()`.
+Note: If you need report an absolute time to an observability tool you can also use `performance.timeOrigin + performance.now()`.
+
+##### Cacheable use cases
+
+If you want to read the time when some cache entry is created such as when a Next.js page is rendered at build-time or when revalidating a static page then move the current time read inside a cached function using `"use cache"`.
 
 Before:
 
 ```jsx filename="app/page.js"
+async function InformationTable() {
+  "use cache"
+  const data = await fetch(...)
+  return <section>
+    <h1>Latest Info...</h1>
+    <table>
+      {renderData(data)}
+    </table>
+  </section>
+}
```

Comment:
I think this reads a bit better breaking the render line up for consistent indentation
```suggestion
async function InformationTable() {
  "use cache"
  const data = await fetch(...)
  return (
    <section>
      <h1>Latest Info...</h1>
      <table>{renderData(data)}</table>
    </section>
  )
}
```